flaky-tests/frameworks/pytest.md

```diff
@@ -14,28 +14,25 @@ layout:
     visible: true
 ---
 
-# Pytest
-
-Pytest is a testing framework for JavaScript and TypeScript.
-
-## Enabling XML Output
+Pytest is a testing framework for Python.
```

Comment:
Is it Pytest or PyTest. See title as "PyTest"

Reply:
The pytest website says `pytest` with no caps, so I'll switch to that everywhere.